easybuild/tools/modules.py

```diff
@@ -1413,6 +1421,7 @@ class Lmod(ModulesTool):
     REQ_VERSION = '6.5.1'
     DEPR_VERSION = '7.0.0'
     REQ_VERSION_DEPENDS_ON = '7.6.1'
+    REQ_VERSION_LUA_CHECK_GROUP = '6.0.8'
```

Comment:
Does Lmod also support `module-info usergroups`?

Since Tcl syntax can also be used when using Lmod as modules tool, we should also define `REQ_VERSION_TCL_CHECK_GROUP` here?

Reply:
Lmod does not support `module-info usergroups` currently. But if this command is added in the future, it will be easy to update EasyBuild to generate modulefile using this command.